pkgs/applications/audio/magnetophonDSP/ConstantDetuneChorus/default.nix

```diff
@@ -29,5 +33,7 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/magnetophon/constant-detune-chorus";
     license = lib.licenses.gpl3;
     maintainers = [ lib.maintainers.magnetophon ];
+    # ERROR3 : n is NaN in an Interval
+    broken = true;
```

Comment:
Remove, maybe? Or will this be fixed in the future?

Reply:
This is working again in the new faust version: https://github.com/NixOS/nixpkgs/pull/231700.